samples/consuming-tas-utility/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,37 @@
+# Set root logger level to error
+#log4j.rootLogger=${app.log.root.level}, Console, File
+rootLogger.level=INFO
+rootLogger.appenderRef.stdout.ref=ConsoleAppender
+rootLogger.appenderRef.rolling.ref=RollingAppender
+
+###### Console appender definition #######
+# All outputs currently set to be a ConsoleAppender.
+# Direct log messages to stdout
+appender.ConsoleAppender.type=Console
+appender.ConsoleAppender.name=ConsoleAppender
+appender.ConsoleAppender.layout.type=PatternLayout
+# use log4j NDC to replace %x with tenant domain / username
+appender.ConsoleAppender.layout.pattern=%d{ISO8601} %x %-5p [%c{3}] [%t] %replace{%m}{[\r\n]+}{}%n
+#log4j.appender.Console.layout.ConversionPattern=%d{ABSOLUTE} %-5p [%c] %m%n
+
+###### File appender definition #######
+# Direct log messages to a log file
+appender.RollingAppender.type=RollingFile
+appender.RollingAppender.name=RollingAppender
+appender.RollingAppender.fileName=./target/reports/alfresco-tas.log
+appender.RollingAppender.filePattern=./target/reports/alfresco-tas.log.%d{yyyy-MM-dd}
+appender.RollingAppender.layout.type=PatternLayout
+appender.RollingAppender.layout.pattern=%d{yyyy-MM-dd} %d{ABSOLUTE} %-5p [%c] [%t] %replace{%m}{[\r\n]+}{}%n
+appender.RollingAppender.policies.type = Policies
+appender.RollingAppender.policies.time.type = TimeBasedTriggeringPolicy
+appender.RollingAppender.policies.time.interval = 1
+
+# TestRail particular log file
+# Direct log messages to a log file
+appender.TestRollingAppender.type=RollingFile
+appender.TestRollingAppender.name=TestRollingAppender
+appender.TestRollingAppender.fileName=./target/reports/alfresco-testrail.log
+appender.TestRollingAppender.layout.type=PatternLayout
+appender.TestRollingAppender.layout.pattern=%d{HH:mm:ss} %-5p %c{1}:%L - %m%n
+appender.TestRollingAppender.strategy.type = DefaultRolloverStrategy
+appender.TestRollingAppender.strategy.max = 10
```

Comment:
This configuration should be largely similar to the one I just proposed earlier as a literal translation of the previous implementation, with minor differences in the file name, appender name, layout pattern structure.